AnbProduct.php

```diff
@@ -352,6 +352,53 @@ function generateServiceDetail($product)
         return $featuresHtml;
     }
 
+    function generateServiceDetailTelecom($product)
+    {
+        $featuresHtml     = '';
+        $lastFeatureHTML = null;
+        $priorityChecked  = false;
+
+        if (isset($product->packtypes)) {
+
+            $product->core_features->internet;
```

Comment:
?

Reply:
Dat is code van de originele functie. Wanneer het geen package deal is, maar alleen internet.

Reply:
Ja maar deze regel doet toch helemaal niks?

Reply:
Klopt, ik zat verkeerd te kijken. Ik keek te ver. Dit had ik over het hoofd gezien